SECURITY.rst

```diff
@@ -0,0 +1,26 @@
+Security Policy
+===============
+
+Supported Versions
+------------------
+
++---------+-----------+
+| Version | Supported |
++=========+===========+
+| 3.7     | Yes       |
```

Comment:
@vanillajonathan FTR, a lot of projects maintain their own pages with docs so the best thing to do is to redirect people there instead of forcing maintainers to watch out for multiple places to update...

btw, are you sure that RST works for this specific file? I wasn't able to locate this in docs

Reply:
Yes, reStructuredText works for the security file on GitHub.
Example: https://github.com/apache/airflow/security/policy

Reply:
Actually, it is a shared part of both Markdown and ReST syntax.
The link to devguide page is preferred anyway for sake of keeping Python versions status information in the single place.

Reply:
Andrew, I think it's a good idea to keep all files with one extension for the sake of consistency

Reply:
I don't care.
Ok with committed code.